.idea/workspace.xml

```diff
@@ -66,6 +223,8 @@
   </component>
   <component name="VcsManagerConfiguration">
     <MESSAGE value="Initial commit" />
-    <option name="LAST_COMMIT_MESSAGE" value="Initial commit" />
+    <MESSAGE value="Add all necessary libraries which are going to be used" />
+    <MESSAGE value="Comment update dependency graph command in maven.yml file." />
+    <option name="LAST_COMMIT_MESSAGE" value="Comment update dependency graph command in maven.yml file." />
   </component>
 </project>
```

Comment:
Hi! I would suggest to add this file to .gitignore

Reply:
Good idea Katsiaryna, thanks. Done 👍